geometry_msgs/Quaternion value
---
int32 success